gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/ITGcsNio.java

```diff
@@ -0,0 +1,351 @@
+package com.google.gcloud.storage.contrib.nio;
```

Comment:
That's a great idea! I've done that. It did indeed catch a problem, as CloudStorageConfiguration.DEFAULT was used and it was package-internal. I've made it public.